Singapore/src/main/java/com/infoshareacademy/service/ReadFile.java

```diff
@@ -0,0 +1,27 @@
+package com.infoshareacademy.service;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class ReadFile {
+
+
+    private String filePath = "src/main/resources/Gdańsk/Monuments/próba.txt";
+    // TODO Stworzyć metodę na generowanie ścieżki
+    public String readFile() {
+        File file = new File(filePath);
+        Scanner in = null;
+        String zdanie = "";
+        try {
+            in = new Scanner(file);
+            zdanie = in.nextLine();
+
+            System.out.println(zdanie);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+        return zdanie;
```

Comment:
Twoim zadaniem było przygotowanie menu. Nie tworzenie logiki czytania plików.

Reply:
To pozostałość błędu z przeszłości, rozgałęziłem się na nowe zadania z innego brancha niż develop, co później pullował Maciej. Musimy te początkowe błędy pchnąć, żeby zacząć sensownie tworzyć projekt. 
Niestety na razie mamy trochę chaos, ale myślę, że uporządkujemy wszystko.